### PR TITLE
feat: support colors in mfe headers

### PR DESCRIPTION
**Pull request by navinkarkera**
See #35 for details.

`Private-ref`: [BB-8635](https://tasks.opencraft.com/browse/BB-8635)

### CODE REVIEW

File: common/_variables.scss [hunk 1]
```diff
@@ -3,6 +3,10 @@ $main-color: #ff8000 !default;
 $link-color: #126f9a !default;
 $header-bg: #fff !default;
 $footer-bg: #fff !default;
+$header-btn-bg: $header-bg !default;
```
Comment (kaustavb12) [inline common/_variables.scss:7]: Nit: a white space between the other variables and header variables for better readability
```suggestion

$header-btn-bg: $header-bg !default;
```

File: paragon/_overrides.scss [hunk 2]
```diff
@@ -144,6 +181,24 @@ footer.footer {
   }
 }
 
+// Discussions mfe overwrites learning header color to white
+.learning-header {
+  background-color: $header-bg !important;
+  .course-title-lockup {
+    // Update course name color in mfe header
+    color: $main-nav-color;
```
Comment (kaustavb12) [inline paragon/_overrides.scss]: `main-nav-color` is optional and we don't define a default for this. So this needs to be move under the [if block here](https://github.com/open-craft/edx-simple-theme/blob/817f97d2421f92602cedff98539d9f1cb2898268/paragon/_overrides.scss#L21-L25)

Infact, do we need to define css for `.course-title-lockup` at all ? Doesn't `header span` which is defined there already cover it ?

File: common/_variables.scss [hunk 3]
```diff
@@ -3,6 +3,10 @@ $main-color: #ff8000 !default;
...
+$header-btn-bottom-border-width: 4px !default;
+$header-btn-bottom-border-hover-color: theme-color("inverse") !default;
+$header-course-link-btn-bottom-border-color: $link-color !default;
```
Comment (kaustavb12) [inline common/_variables.scss:10]: Please add the new variable in the [README](https://github.com/open-craft/edx-simple-theme/blob/master/README.md)

**Review by kaustavb12 — CHANGES_REQUESTED** [review]

**Review by kaustavb12 — APPROVED** [review]
:+1:

- [x] I tested this: Tested that issues have been fixed
- [x] I read through the code
- [ ] Includes documentation

File: paragon/_overrides.scss (hunk 2 above)
Comment (navinkarkera) [reply paragon/_overrides.scss]: Yes. The `header span` change was missing from Olive and I was using it as base initially which led to this confusion. Removed this part!

**Pull request merged**